bot.js

```diff
@@ -117,13 +117,22 @@ const stream = bot.voice.createBroadcast();
 // This variable will track our current stream status, for reporting reasons
 let streamStatus = "Startup";
 
+// This will be the path we connect to.
+// By default, we shall use fallback paths set in config
+// But, if configured to try to auto-fetch it, we'll grab the new path and put it here.
```

Comment:
I considered also adding behavior for the fallback path to get used later if the stream fails to connect or dies - As a hardcoded redundancy, not just a block to the listenURL not getting updated.

I spent about 10 seconds on it and didn't think of a great way to differentiate a temporary network issue from a condition where we should set back to fallback, except implementing a reconnect counter and tuning it for some big number, which seems sad.

Thoughts? Would this be expected behavior I should put in?